tests/test_keras_integration.py

```diff
@@ -228,6 +228,26 @@ def test_save_pretrained_model_card_fit(self):
         self.assertIn("history.json", files)
         self.assertEqual(len(files), 7)
 
+    def test_save_model_card_history_removal(self):
+        REPO_NAME = repo_name("save")
```

Comment:
not related to this PR, but `repo_name` seems to be the idea that it's a random repo name, but passing a fixed string makes it less random and creates the randomness from the `time`, which is not really what we want. Not passing any argument here would be a safer option.